lib/appliance_console/database_configuration.rb

```diff
@@ -173,6 +173,11 @@ def self.current
       decrypt_password(load_current)
     end
 
+    def self.configured?
+      db_yml = RAILS_ROOT.join("config/database.yml")
+      File.exist?(db_yml)
```

Comment:
So, we have this path elsewhere, right?  Can we make this a constant in this class and do File.exist?(DATABASE_YML) in this method and fix the other places to use this constant?

See: https://github.com/ManageIQ/manageiq/blob/73e8ac75e8f5a145819ad389791299f46ea9a962/lib/appliance_console/database_configuration.rb#L209-L220